### ISSUE TITLE
Run mksurfdata_esmf with the new vegetation raw data

### ISSUE THREAD

**Issue by slevis-lmwg** [ESCOMP/CTSM#2452]
@lawrencepj1 has a new set of raw datasets for vegetation that we need to test with mksurfdata_esmf

In `/glade/campaign/cesm/development/lmwg/landuse_source_data/CTSM5.2RawData` there are 10 directories:
```
globalctsm52CCISENSBASELINEDeg025_240402  globalctsm52SSP126Deg025_231228
globalctsm52histLUH2Deg025_231228         globalctsm52SSP245Deg025_231228
globalctsm52histMKSRFDeg025_231228        globalctsm52SSP370Deg025_231228
globalctsm52histTRENDY2023Deg025_231228   globalctsm52SSP460Deg025_231228
globalctsm52SSP119Deg025_231228           globalctsm52SSP585Deg025_231228
```
In the above,
MKSRF contains the lai and soilcolor raw data
no need to test the contents of CCISENSBASELINE and TRENDY

- [x] @slevis-lmwg @lawrencepj1 @olyson @wwieder tested successfully and with satisfactory results (https://github.com/NCAR/LMWG_dev/issues/59) using the above raw datasets, but we have decided instead to go with the TRENDY2023 and soon TRENDY2024 data.
- [x] @slevis-lmwg test mksurfdata_esmf with f09.
- [x] @slevis-lmwg test with f09 offline simulations compared to current ctsm5.2 datasets (control):
1850 spinup and historical (https://github.com/NCAR/LMWG_dev/issues/59)
SSP245 (didn't do)
- [x] Make all fsurdat/landuse files; name them based on the branch tag used to create them (WON'T DO as part of this issue).

**Comment by slevis-lmwg** [comment]
@wwieder pinging you to make you aware of this discussion with @lawrencepj1 and @ekluzek

**Comment by slevis-lmwg** [comment]
I'm adding my part of the work to the Sprint board.
(Mentioning this because the issue is assigned to me and @lawrencepj1.)

**Comment by lawrencepj1** [comment]
Thanks @slevis-lmwg @ekluzek and @wwieder

The new datasets have improved cropping distributions as well as the capability for reforestation and bioenergy with carbon capture and storage (although the carbon capture will need further work). The updated crop distributions can be seen in the attached power point slides.

https://docs.google.com/presentation/d/1886QjkvUfYFAZRB-LPykir2We568ah1-/edit?pli=1#slide=id.p1

Cheers
Peter

**Comment by lawrencepj1** [comment]
Thanks Sam and Will
-- 
Dr Peter Lawrence
Terrestrial Science Section
National Center for Atmospheric Research
1850 Table Mesa Drive
Boulder Colorado 80305

Work: 1-303-497-1727
Cell: 1-303-956-6932

**Comment by slevis-lmwg** [comment]
@lawrencepj1 I don't remember whether we discussed this already:
In the above directories, is there a replacement file for the noantro "PtVg" case? Or should I not worry about that for now?

**Comment by slevis-lmwg** [comment]
Generated f09 1850 fsurdat and 1850-2100 landuse files for all the SSPs listed above. They are here:
`/glade/work/slevis/git/mksurfdata_toolchain/tools/mksurfdata_esmf`
I will next confirm whether the model runs successfully with them.

**Comment by slevis-lmwg** [comment]
@lawrencepj1 I started a simulation (described in prev. post) that seems to be working, so I think my TODOs are complete in this github issue.

**Comment by lawrencepj1** [comment]
Thanks @slevis-lmwg 

This is great. In terms of the PtVg case I don't know how that works. Excellent to hear that the updated files all ran through mksurfdata_esmf and the model ran without issue. Next steps will be to look at the historical simulations once completed. Are you intending to let it run to 2100 under the SSP245 compset?

Cheers
Peter

**Comment by slevis-lmwg** [comment]
Based on the notes that we took at the top of the page, I'm letting you run the simulations that you needed. I hope you're still fine with that plan. Feel free to clone my test case from here: ~/cases_ctsm5.3/issp245clm53bgccrop_f09

Regarding PtVg: I'm realizing that your updates do not affect PtVg, so we will continue to point to the existing noanthro file:
`lnd/clm2/rawdata/pftcftdynharv.0.25x0.25.LUH2.noanthro.c20230226/mksrf_landuse_ctsm52_noanthroLUH2_1.c20230226.nc`

**Comment by lawrencepj1** [comment]
Thanks @slevis-lmwg 

Yes I can take a look at the case and run it from there.

Cheers
Peter

**Comment by slevis-lmwg** [comment]
From meeting with @lawrencepj1 @ekluzek @mvdebolskiy 
- @slevis-lmwg will continue and finish the f09 historical to 2015
- @slevis-lmwg will clone the case to run the same with the 5.2 data
- We will ask Keith Oleson to run the diag. package

**Comment by slevis-lmwg** [comment]
Reminder that this issue fits in the broader context of
https://github.com/ESCOMP/CTSM/discussions/2206

Also this issue is partly addressed by #2500

**Comment by slevis-lmwg** [comment]
This work has evolved beyond this issue and now appears in #2500.
